public/partials/report/modals/filter-prompt-modal.html

```diff
@@ -32,6 +32,35 @@ <h4 class="modal-title" translate>Runtime Filter properties</h4>
                 </div>
             </div>
         </div>
+
+        <div class="row" ng-if="selectedFilter.elementType !== 'date'">
+
+            <div class="col-md-8">
+                <label class="control-label" translate>Suggestions</label>
+                <div class="popover-primary ng-scope" popover-trigger="mouseenter" popover-placement="top" popover="{{ 'Select if you want suggest values on filter' | translate }}"></div>
+                    <div class="checkbox">
+                        <label>
+                            <input type="checkbox" ng-model="selectedFilter.suggestions.show">
+                            <span translate>Show suggestions</span>
+                        </label>
+                    </div>
+                    <div class="checkbox" ng-if="selectedFilter.suggestions.show && !selectedFilter.suggestions.unlimited">
+                        <label>
+                            <span translate>Number of suggestions</span> 
+                            <input type="number" ng-model="selectedFilter.suggestions.limit" placeholder="{{selectedFilter.suggestions.limit}}">
+                        </label>
+                    </div>
+                </div>
+            </div>
+            <div ng-if="selectedFilter.suggestions.show && selectedFilter.elementType !== 'date'" class="alert alert-warning col-md-12 text-justify">
```

Comment:
There is already a check for the element type on the container div (line 36). I think it's not needed here.